### ISSUE TITLE
match on create does not seem to be able to be created with the QueryBuilder

### ISSUE THREAD

**Issue by Eforen** [themetalfleece/neogma#19]
I could  be wrong but there does not seem to be a way to build an [ON CREATE ](https://neo4j.com/docs/cypher-manual/current/clauses/merge/#query-merge-on-create-on-match) query like this 
```cypher
MERGE (keanu:Person {name: 'Keanu Reeves'})
ON CREATE
  SET keanu.created = timestamp()
RETURN keanu.name, keanu.created
```
This example taken from the [Neo4j Docs](https://neo4j.com/docs/cypher-manual/current/clauses/merge/#query-merge-on-create-on-match)

**Comment by themetalfleece** [comment]
Correct. This cannot be built by using only build-in operators, but it can be built by using `raw`.
For example:
```ts
.merge(...)
.raw(`ON CREATE SET keanu.created = timestamp()`)
.return([...]);
```

**Comment by Eforen** [comment]
Good to know thats better then just using raw but would be good to be able to use native
